### PR TITLE
Add `zcrypt` adapter specification

### PR DESCRIPTION
**Pull request by kkaarreell**
Pull Request Checklist

* [x] update the specification
* [x] modify the json schema
* [x] include a release note

### CODE REVIEW

**Review by psss — APPROVED** [review]
Thanks for extending the spec!

**Comment by psss** [comment]
Added a short release note in c78e8f9a.

File: spec/hardware/zcrypt.fmf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+description: |
+    .. code-block::
+
+       zcrypt:
+           # String, adapter requested.
+           adapter: "x"
+           # String, mode requested.
+           mode: "y"
+
```
Comment (happz) [inline spec/hardware/zcrypt.fmf:12]:
```suggestion

.. versionadded:: 1.31

```

**Review by happz — APPROVED** [review]

File: spec/hardware/zcrypt.fmf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+.. versionadded:: 1.31
```
Comment (happz) [inline spec/hardware/zcrypt.fmf:13]:
```suggestion
    .. versionadded:: 1.31
```

@psss sorry, needs to be indented :/

**Pull request merged**